bot/src/db.rs

```diff
@@ -86,6 +101,12 @@ pub struct EpochClassificationV1 {
 
     // Informational notes regarding this epoch
     pub notes: Vec<String>,
+
+    // Config values from Config struct
+    pub config: Option<Value>,
+
+    // General info about the Epoch
+    pub info: Option<Mapping>,
```

Comment:
The dynamic typing for `config` and `info` seems like it'll make it more likely for the downstream program to break unintentionally due to changes in this file. Some future cleanup/refactoring of the `Config` struct for example could easily break the downstream processing. Your call if you want to do this, I feel a more robust path would be to define a new `Mapping` struct that calls out each field, and also includes the fields in `Config` that are interesting for the downstream.

Reply:
Yeah, I agree. I've changed these into Structs. The Struct for the config is all Options, with the reasoning that some configuration variables might get removed at some point.

I've also fixed the tests that I broke.